library/cwm/src/modules/CWM.rb

```diff
@@ -1061,11 +1065,10 @@ def validate_current_widgets(event)
     publish function: :InitNull, type: "void (string)"
     publish function: :StoreNull, type: "void (string, map)"
 
-  private
-
     def widgets_in_contents(contents)
       contents.each_with_object([]) do |arg, res|
         case arg
+        when ::CWM::CustomWidget then res.concat(arg.nested_widgets) << arg
```

Comment:
your `else` cannot work, as in content is a lot of other objects like strings, numbers, etc. So we need detect term and do nesting.
Regarding widgets it is possible to add it.